example/generate.py

```diff
@@ -0,0 +1,71 @@
+from sys import version_info, path
+from os.path import join, abspath, dirname
+
+path.insert(0, abspath(join(dirname(__file__), "..")))
+
+from novelai_api import NovelAI_API
+from novelai_api.utils import (
+    get_encryption_key,
+    decrypt_user_data,
+    map_meta_to_stories,
+    assign_content_to_story,
+)
+from aiohttp import ClientSession
+
+from logging import Logger
+from asyncio import run
+
+import json
+
+# pip3 install pyyaml
+import yaml
+
+filename = join("credentials", "creds_example.txt")
+with open(filename) as f:
+    username, password = f.read().split(",")
+
+logger = Logger("NovelAI")
+
+
+async def main():
+    async with ClientSession() as session:
+        api = NovelAI_API(session, logger=logger)
+
+        print("login..")
+        login = await api.high_level.login(username, password)
+
+        print("get_encryption_key..")
+        key = get_encryption_key(username, password)
+        keystore = await api.high_level.get_keystore(key)
+        print(keystore)
+
+        print("generate..")
+        input = "sLTgAMYAAgARDt4CIgHeAsYA"
```

Comment:
This input looks like a compressed token list, like what you see in the generate requests.
It shouldn't be given directly. Now, either a string of clear text or a list of tokens should be passed.
Either way, it would have been interpreted as clear text, since the "use_string" key was True in the generate endpoint.

Reply:
Yep. That was in an earlier commit, showing the evolution of the code.
The later commit generates the base64 input (in to demonstrate how) and also uses the plain text input per "use_string".